snmp/datadog_checks/snmp/data/profiles/hp-ilo4.yaml

```diff
@@ -13,6 +14,14 @@ device:
 
```

Comment:
Could you remove:

```
device:
  vendor: "hp"
```

for all profiles extending `_hp-base.yaml` that is not needed anymore since we have:

```
metadata:
  device:
    fields:
      vendor:
        value: "hp"
```

Reply:
Can't as it's relied upon for metric tags